docs/guide/rpc-api.md

```diff
@@ -1,5 +1,309 @@
+}
+
+ethereum.request({
+  method: 'wallet_requestPermissions',
+  params: [ RequestedPermissions ]
+}): Promise<Web3WalletPermission[]>
+```
+
+Requests the given permissions from the user.
+Returns a Promise that resolves to a non-empty array of `Web3WalletPermission` objects, corresponding to the caller's current permissions.
+If the user denies the request, the Promise will reject with a `4001` error.
+
+The request causes a MetaMask popup to appear.
+You should only request permissions in response to user action, such as a button click.
+
+#### Example
+
+```javascript
+document.getElementById('requestPermissionsButton', requestPermissions);
+
+function requestPermissions() {
+  ethereum
+    .request({
+      method: 'eth_requestAccounts',
+      params: [{ eth_accounts: {} }],
+    })
+    .then((permissions) => {
+      const accountsPermission = permissions.find(
+        (permission) => permission.parentCapability === 'eth_accounts'
+      );
+      if (accountsPermission) {
+        console.log('eth_accounts permission successfully requested!');
+      }
+    })
+    .catch((error) => {
+      if (error.code === 4001) {
+        // EIP-1193 userRejectedRequest error
+        console.log('Permissions needed to continue.');
+      } else {
+        console.error(error);
+      }
+    });
+}
+```
+
+### wallet_getPermissions
+
+```typescript
+ethereum.request({
+  method: 'wallet_requestPermissions',
+}): Promise<Web3WalletPermission[]>;
+```
+
+Gets the caller's current permissions.
+Returns a Promise that resolves to an array of `Web3WalletPermission` objects.
+If the caller has no permissions, the array will be empty.
+
+## Other RPC Methods
+
+### wallet_registerOnboarding
+
+::: tip
+As an API consumer, you are unlikely to have to call this method yourself.
+Please see [`@metamask/onboarding`](https://github.com/MetaMask/metamask-onboarding) for more information.
+:::
+
+```typescript
+ethereum.request({ method: 'wallet_registerOnboarding' }): Promise<true>;
+```
+
+Requests that MetaMask completes its onboarding flow for the user.
+Returns a Promise that resolves to `true`, or rejects if there's an error.
+
+This method is intended to be called at the end of the MetaMask onboarding flow.
+The onboarding flow is initiated by our [`@metamask/onboarding`](https://github.com/MetaMask/metamask-onboarding) package, which calls `wallet_registerOnboarding` upon redirecting the user back to the original site.
+Instead of calling this method directly, you should use [`@metamask/onboarding`](https://github.com/MetaMask/metamask-onboarding) directly.
+
+### wallet_watchAsset
+
+::: tip EIP-747
+This method is specified by [EIP-747](https://eips.ethereum.org/EIPS/eip-747).
+:::
+
+```typescript
+interface WatchAssetParams {
+  type: 'ERC20', // In the future, other standards will be supported
+  options: {
+    address: string, // The address of the token contract
+    'symbol': string, // A ticker symbol or shorthand, up to 5 characters
```

Comment:
Why quote just this property :thinking: 
```suggestion
    symbol: string, // A ticker symbol or shorthand, up to 5 characters
```

Reply:
Because `symbol` is a reserved keyword, and the syntax highlighting picks it up.

Not sure if it'd be better to quote everything?

Reply:
Ah, no, this seems fine. I totally forgot it was a reserved keyword.